sdk/servicebus/Azure.Messaging.ServiceBus/src/Processor/ServiceBusProcessorOptions.cs

```diff
@@ -38,26 +39,28 @@ public int PrefetchCount
 
         /// <summary>
         /// Gets or sets the <see cref="ReceiveMode"/> used to specify how messages
-        /// are received. Defaults to PeekLock mode.
+        /// are received.
         /// </summary>
+        ///
+        /// <value>The mode to use for receiving messages. The default value is <see cref="ServiceBusReceiveMode.PeekLock"/>.</value>
         public ServiceBusReceiveMode ReceiveMode { get; set; } = ServiceBusReceiveMode.PeekLock;
 
         /// <summary>
         /// Gets or sets a value that indicates whether the processor
         /// should automatically complete messages after the <see cref="ServiceBusProcessor.ProcessMessageAsync"/> handler has
         /// completed processing. If the message handler triggers an exception, the message will not be automatically completed.
-        /// The default value is true.
         /// </summary>
         ///
-        /// <value>true to complete the message automatically on successful execution of the message handler; otherwise, false.</value>
+        /// <value>true to complete the message automatically on successful execution of the message handler; otherwise, false.
```

Comment:
```suggestion
        /// <value><c>true</c> to complete the message automatically on successful execution of the message handler; otherwise, <c>false</c>.
```

The nittest of nits, but we're using the code element on the next line, so it seemed good to be consistent.